core/src/main/java/com/scalar/db/storage/cassandra/Cassandra.java

```diff
@@ -173,7 +173,9 @@ public void mutate(List<? extends Mutation> mutations) throws ExecutionException
 
     Utility.setTargetToIfNot(mutations, namespacePrefix, namespace, tableName);
     operationChecker.check(mutations);
-    mutations.forEach(operationChecker::check);
+    for (Mutation mutation : mutations) {
```

Comment:
Any particular reason for not using `forEach`?

Reply:
Thank you for taking at this!

After this change, `operationChecker::check` throws `ExecutionException`.  `forEach` doesn't handle Exception well, I changed this way.